CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+v0.5.2 - February 2 2016
+
+* "expect" function adds third output, the input string which resulted in the match
```

Comment:
"expect" function adds third parameter to callback, the input string which resulted in the match